src/backend/linux/window.rs

```diff
@@ -677,6 +677,20 @@ impl WindowHandle {
             WindowHandle::None => panic!("Used an uninitialised WindowHandle"),
         }
     }
+
+    #[cfg(feature = "accesskit")]
+    pub fn update_accesskit_if_active(
+        &self,
+        update_factory: impl FnOnce() -> accesskit::TreeUpdate,
+    ) {
+        match self {
+            #[cfg(feature = "x11")]
+            WindowHandle::X11(handle) => handle.update_accesskit_if_active(update_factory),
+            #[cfg(feature = "wayland")]
+            WindowHandle::Wayland(handle) => handle.update_accesskit_if_active(update_factory),
+            WindowHandle::None => panic!("Used an uninitialised WindowHandle"),
```

Comment:
I was going to comment on this being ``uninitialised` when everything else uses `uninitialized` in Glazier (including API), except for these panic messages.

Reply:
I mean, you're right. But it doesn't spark joy.

I don't think changing it in this PR makes sense, as ~~correct~~ British spellings are used all across the new Wayland backend.